tests/primitives/test_python_interpreter.py

```diff
@@ -1,4 +1,10 @@
-from dspy.primitives.python_interpreter import PythonInterpreter
+import shutil
+import pytest
+from dspy.primitives.python_interpreter import PythonInterpreter, InterpreterError
+
+# This test suite requires deno to be installed. Please install deno following https://docs.deno.com/runtime/getting_started/installation/
+if shutil.which("deno") is None:
+    pytestmark = pytest.mark.skip(reason="Deno is not installed or not in PATH")
```

Comment:
Let's use this instead of the magic global var `pytestmark` for clarity:

```
if shutil.which("deno") is None:
    pytest.skip("Deno is not installed or not in PATH", allow_module_level=True)
```